tests/StatsdServer.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef STATSD_SERVER_HPP
+#define STATSD_SERVER_HPP
+
+// TODO: should we actually add this to the distributed headers? With a tiny bit of work
+//  someone could write their own c++ statsd server
```

Comment:
should i move this to the include directory so that others can use it, i will personally use this in my project since it lets me validate that im sending the stats that i expect to.